slsDetectorServers/mythen3DetectorServer/slsDetectorFunctionList.c

```diff
@@ -1468,14 +1555,22 @@ u_int32_t runBusy() {
     return virtual_status;
 #endif
 	u_int32_t s = (bus_r(PAT_STATUS_REG) & PAT_STATUS_RUN_BUSY_MSK);
-	FILE_LOG(logDEBUG1, ("Status Register: %08x\n", s));
+	//FILE_LOG(logDEBUG1, ("Status Register: %08x\n", s));
 	return s;
 }
 
 /* common */
 
 int calculateDataBytes() {
-	return 0;
+	int numCounters = __builtin_popcount(getCounterMask());
```

Comment:
There is no 8 bit mode?

I'm also not sure that this works for 1 bit. Seems to round down for me...

Reply:
Nope, only 1, 4 16 and 24. I will look into it once I allow them to change dynamic range in the server.